src/scss/base.scss

```diff
@@ -215,7 +224,10 @@ body {
   padding: 60px 20px;
 
   .logo {
-    margin-bottom: 40px;
+    margin-bottom: 20px;
+    width: 172px;
```

Comment:
just set one of those and let the otherone automatically adjust. The image might deform